examples/selenium-container/src/main/resources/templates/foo.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
```

Comment:
I think you can move it to `src/main/resources/static/foo.html` and simply query it with `/foo.html`, so that you don't need to create a controller

Reply:
ok, changed